packages/utils/src/webpack/extract/assets.js

```diff
@@ -24,11 +24,13 @@ export const extractAssets = (webpackStats) => {
     .flat();
 
   const assets = webpackAssets.reduce((aggregator, asset) => {
-    if (IGNORE_PATTERN.test(asset.name)) {
+    const baseName = asset.name && asset.name.split('?')[0];
```

Comment:
I was referring before the [pattern replacement inside `getAssetName`](https://github.com/relative-ci/bundle-stats/blob/master/packages/utils/src/webpack/utils.js#L42), but is probably better here since it can ignore if is matching `IGNORE_PATTERN` ;)